docker-compose.yml

```diff
@@ -26,3 +26,9 @@ services:
             - ./api/src:/api/src
         ports:
             - 3000:3000
+    he_twitter_miner:
+        build: .\he_twitter_miner
+        volumes:
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\he_twitter_miner\src:/usr/src/app
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\config:/usr/src/app/config
```

Comment:
you might be right on this one, i can switch the directions of the slashes. is that what you mean?